bigtable/run_hbase_commands.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python
+
+import subprocess
+
+
+def create_commands_file():
+    file = open("commands", 'w+')
```

Comment:
Nit: `with open(...) as file` (and then you don't need `file.close()`)

Reply:
using `with open` make a lot of sense when dealing with files. Thanks.